apps/site/assets/css/app.scss

```diff
@@ -7,7 +7,10 @@
 @import 'fonts';
 @import 'navbar';
 @import 'buttons-overrides';
-//
+$link-decoration: none;
```

Comment:
This was already set in the [fonts.scss](https://github.com/mbta/dotcom/blob/53cb06b630bc539e79299af4956377f3170c383c/apps/site/assets/css/_fonts.scss#L20) file.

Reply:
Ah, yeah let's consolidate those overrides.